app/views/admin/members/payment_whatsapp.html.haml

```diff
@@ -0,0 +1,7 @@
+= I18n.t('admin.payment.whatsapp.title', user: current_user.credentials.first_name)
```

Comment:
Very nice to see that we can do away with `erb-loader`, that is going to save us some trouble going forward.
Can you also remove the package itself?

Reply:
erb-loader is still being used for I18n-js, not it can't be removed. It generates a js file with the template that includes the translations.